heron/config/src/yaml/conf/local/packing.yaml

```diff
@@ -1,2 +1,3 @@
 # packing algorithm for packing instances into containers
 heron.class.packing.algorithm:    com.twitter.heron.packing.roundrobin.RoundRobinPacking
+heron.class.repacking.algorithm:  com.twitter.heron.packing.binpacking.FirstFitDecreasingPacking
```

Comment:
There are separate interfaces for each (`IPacking` and `IRepacking`) so implementors can implement one or the other. An `IRepacking` impl should not assume anything about what `IPacking` impl was used at submit.